src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -125,18 +131,16 @@ export default class ReserveSubmit extends Vue {
 
     let data: any
     let request: NameRequestI
-    switch (setup) {
+    switch (setup as any) {
```

Comment:
ESLint didn't like the previous "@ts-ignore" comments so I worked around it this way.